x/airdrop/keeper/zonedrop.go

```diff
@@ -98,17 +98,17 @@ func (k Keeper) AllActiveZoneDrops(ctx sdk.Context) []types.ZoneDrop {
 // d: decay     (time.Duration) ... s+D+d
 //
 // isActive:  s <= timenow <= s+D+d
-// notActive: timenow < s || timenow > s+D+d
+// notActive: timenow < s || timenow > s+D+d.
 func (k Keeper) IsActiveZoneDrop(ctx sdk.Context, zd types.ZoneDrop) bool {
 	bt := ctx.BlockTime()
 
 	// negated checks here ensure inclusive range
-	return !bt.Before(zd.StartTime) && !bt.After(zd.StartTime.Add(zd.Duration).Add(zd.Decay))
+	return bt.After(zd.StartTime) && !bt.After(zd.StartTime.Add(zd.Duration).Add(zd.Decay))
```

Comment:
this wont be an issue but its not inclusive of bt == Startime - just pointing the conditions are not same

Reply:
reverted